From 4ec5a1fb8a0ba5b1a52a13c44026f396ca8d82ea Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Tue, 18 Jun 2024 13:59:35 +0000
Subject: [PATCH] Disable libicuuc lib fro A15

This patch will disable libicuuc since it is giving build
errors.

Tests Done: Build should proceed

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 54b431f..0e6b670 100644
--- a/Android.mk
+++ b/Android.mk
@@ -54,7 +54,7 @@ LOCAL_STATIC_LIBRARIES := libxml2
 ifeq ($(BOARD_VNDK_VERSION),current)
 LOCAL_SHARED_LIBRARIES := liblog libcutils libdl libc++ libutils
 else
-LOCAL_SHARED_LIBRARIES := liblog libcutils libdl libc++ libicuuc libutils
+LOCAL_SHARED_LIBRARIES := liblog libcutils libdl libc++ libutils
 endif
 LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE := thermal-daemon
-- 
2.34.1

